### PR TITLE
use environment-markers instead of python-logic

### PR DESCRIPTION
**Pull request by pbauer**
See https://www.python.org/dev/peps/pep-0508/#environment-markers

### CODE REVIEW

**Comment by gforcada** [comment]
@pbauer @mauritsvanrees do wheels respect these markers? i.e. can we turn on again the universal wheel then with this? :thinking:

**Comment by mauritsvanrees** [comment]
I suppose so, but I haven't tried these markers.

**Comment by tisto** [comment]
Nice. Can we try out the markers on another (not so super important) package first maybe? :)

**Comment by pbauer** [comment]
We use these marker in Plone: https://github.com/plone/Plone/blob/master/setup.py
Not not important 😉

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

**Comment by gforcada** [comment]
And if you are worried, just make an alpha release, so that nobody accidentally gets that release.

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 from setuptools import setup, find_packages
 
-import sys
-
-version = '3.7.2-dev.0'
+version = '3.7.1.dev.0'
```
Comment (jensens) [inline setup.py]: Version decrease by accident?

**Comment by jensens** [comment]
Environment markers in its current version are from active [PEP 508](/#environment-markers) but are dating back to a more basic version (like we use them) in accepted [PEP 345](https://www.python.org/dev/peps/pep-0345/#environment-markers). Wheels are designed to respect whatever metadata is provided (see [PEP 427](https://www.python.org/dev/peps/pep-0427)) and package it. Pip reads the metadata and is [responsible to respect](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers) it.

So, yes, this is the way to go. With or without wheel. And best put [all data in a `setup.cfg` (example) ] (https://github.com/collective/collective.es.index/blob/master/setup.cfg) and a one liner in ``setup.py`` to never ever have the idea again in future to use logic for metadata.

File: setup.py (hunk 1 above)
Comment (tisto) [reply setup.py]: Fixed.

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**